docs/dataformat.rst

```diff
@@ -37,13 +37,16 @@ Examples are grouped by intent, and entities are annotated as markdown links.
     ## regex:zipcode
     - [0-9]{5}
 
+    ## lookup:accounts   <!-- lookup table of account names for improving entity extraction (savings, checking, ...) -->
+    - path/to/accounts.txt
```

Comment:
Ok moving the conversation over here. So my concern here is still that this probably isn't a thing you should really use the lookup table for right? Like, this would pick up an entity in "What's  in my checking account" as well as in "I'm just checking" for example. I think we should adjust this whole example to the rest of our NLU examples anyways, which is restaurants. 
Let me know what you think, and then we can discuss in the comments here on how exactly to make this example better

Reply:
I agree that we should have this entire example only about restaurants. Having two different domains for the same example looks a bit too confusing :D

Reply:
ok I made the lookup table 'currency' like (yen, USD, etc.) that should be reasonably unambiguous.  also added an example with a currency entity.  Perhaps a full update to the docs should be done in another PR but I agree that a restaurant example would be more clear.

Reply:
An option could be to combine the crf and phrase matcher output, or at least to provide this option with a 3rd component later in the pipeline, say, `crf_phrase_matcher`. Suppose you want to match airport codes for departures and landing (like roles). The phrase matcher alone won't be very useful. However, if you label "from" and "to" entities as usual and let the phrase, the ner_crf will extract them, the phrase matcher will add 2 airport codes to the output, and the  `crf_phrase_matcher` would remove the from/to entities if the value is not in the "airport_codes" lookup table, and optionally any airport code not paired with a labeled entity

You would then specify this in the config file with something like this: 
```yaml
- ner_crf
- ner_phrase_matcher
- ner_crf_phrase_matcher
   from: airport_codes
   to: airport_codes
```